.github/workflows/submit.yml

```diff
@@ -8,13 +8,11 @@ jobs:
   submit:
     uses: curvenote/actions/.github/workflows/submit.yml@v1
     with:
-      id-pattern-regex: '^morganton-2025-(?:[a-zA-Z0-9-_]{3,15})$'
+      id-pattern-regex: '^morganton-2025-(?:[a-zA-Z0-9-_]{2,15})$'
       venue: ncssm-mor
       collection: '2025'
       kind: original
       path: papers/*
-      label: publish
-      publish: true
```

Comment:
I removed `publish` from here - If present, it will publish a new version to the site without any additional review (as opposed to creating a submission which you then have to publish separately).

If you do want this automatic publishing on `main`, feel free to add this back.